### PR TITLE
Support relative difference plot in zonal mean 2d set

### LINKED ISSUE #636: suggestions on improving pressure-lat zonal mean specific humidity plots
For the pressure-lat zonal mean specific humidity plots, e.g., [this one](https://web.lcrc.anl.gov/public/e3sm/diagnostic_output/ac.zke/20221130.production.NGD_v3atm.chem-only.H2OTEST.amip.chrysalis/e3sm_diags/180x360_aave/model_vs_obs_1985-2014/viewer/zonal_mean_2d/era5/q-global-era5/ann.html). 

- It makes more sense to add the relative difference plot besides the existing absolute difference plot because of the value range of variable "Q". This will help understand its impact on heating rate and temperature. For instance, the [200-hPa T bias](https://web.lcrc.anl.gov/public/e3sm/diagnostic_output/ac.zke/20221130.production.NGD_v3atm.chem-only.H2OTEST.amip.chrysalis/e3sm_diags/180x360_aave/model_vs_obs_1985-2014/viewer/zonal_mean_2d/era5/t-global-era5/djf.html) is largely due to the relative water vapor change at that region, which doesn't show at all on the current plot.
- With the new interactive chemistry for E3SMv3, the water vapor variable passing to radiation will be "H2OLNZ" instead of "Q". It's worth adding another H2OLNZ plot in parallel to the Q plot.

### PR DESCRIPTION
**Pull request by chengzhuzhang**
Close #636 
Supersede PR #637

### CODE REVIEW

File: e3sm_diags/derivations/acme.py [hunk 1]
```diff
@@ -133,6 +133,14 @@ def qflxconvert_units(var):
     return var
 
 
+def w_convert_q(var):
+    if var.units == "mol/mol":
+        var = var / (1.0 + var) * 1000.0  # convert to g/kg
```
Comment (chengzhuzhang) [inline e3sm_diags/derivations/acme.py:138]: @tangq would you check if the unit conversion is correct?

**Review by chengzhuzhang — COMMENTED** [review]
@tangq, I'm a little behind on this PR. But I think it is now ready for review. Would you please check if the additional variables added look okay: https://web.lcrc.anl.gov/public/e3sm/diagnostic_output/zhang40/relative_diff_try2/viewer/zonal_mean_2d/index.html? I'm not exactly sure if we should include obs for H2OLNZ, and if the units conversion I had makes sense. Please comment. Thank you!

File: e3sm_diags/derivations/acme.py (hunk 1 above)
Comment (tangq) [reply e3sm_diags/derivations/acme.py:138]: @chengzhuzhang , you will need the molecular weights of H2O (18 g/mol) and dry air (28.97 g/mol) to convert mol/mol to g/kg.
I notice that some typos (k/kg) of the units on figures like: https://web.lcrc.anl.gov/public/e3sm/diagnostic_output/zhang40/relative_diff_try2/viewer/zonal_mean_2d/merra2/h2olnz-global-merra2/ann.html

**Comment by tangq** [comment]
I think we should include obs on the H2OLNZ plot to have some reference. Are we going to have the relative difference plots too? Thanks adding these figures.

File: e3sm_diags/derivations/acme.py [hunk 2]
```diff
@@ -133,6 +133,15 @@ def qflxconvert_units(var):
     return var
 
 
+def w_convert_q(var):
+    if var.units == "mol/mol":
+        var = var * 18.0 / 28.97  # convert to mixing ratio
```
Comment (chengzhuzhang) [inline e3sm_diags/derivations/acme.py:138]: It's an embarrassing miss...I correct the formula. Now values of Q and H2OLNO are very close, not identical though.

**Comment by chengzhuzhang** [comment]
> I think we should include obs on the H2OLNZ plot to have some reference. Are we going to have the relative difference plots too? Thanks adding these figures.

Thank you for reviewing and catching the error. It is now fixed, hopefully. Please check again: https://web.lcrc.anl.gov/public/e3sm/diagnostic_output/zhang40/relative_diff_try3/viewer/zonal_mean_2d/index.html. If things look okay, I will add the relative difference plots of both variables into default variables list.

**Comment by chengzhuzhang** [comment]
@tangq this PR should be ready for merge with your approval.

**Comment by tangq** [comment]
Thanks, @chengzhuzhang . The PR can be merged after the minor changes suggested above are resolved.

**Comment by chengzhuzhang** [comment]
> Thanks, @chengzhuzhang . The PR can be merged after the minor changes suggested above are resolved.

@tangq yes it has been resolved: https://web.lcrc.anl.gov/public/e3sm/diagnostic_output/zhang40/relative_diff_try3/viewer/zonal_mean_2d/index.html And I added H2OLNZ to default variable list with both relative and absolute difference plot.

**Comment by tangq** [comment]
@chengzhuzhang , these two minor ones seem still pending.

- var = var / (1.0 + var) * 1000.0. Why not var = var * 1000.0?
- revise the comment to "convert to mass mixing ratio". The unit "mol/mol" is volume mixing ratio.

**Comment by chengzhuzhang** [comment]
> @chengzhuzhang , these two minor ones seem still pending.
> 
>     * var = var / (1.0 + var) * 1000.0. Why not var = var * 1000.0?
> 
>     * revise the comment to "convert to mass mixing ratio". The unit "mol/mol" is volume mixing ratio.

Got it! I think your last round of comment went missing. I will add "mass" to differentiate from volume mixing ratio. The formula `var = var / (1.0 + var) *1000` is to convert to specific humidity to match Q. But I think the difference can be ignored since the variable is very small.

**Review by tangq — APPROVED** [review]
Thank you for working on this, @chengzhuzhang

File: e3sm_diags/derivations/acme.py [hunk 3]
```diff
@@ -133,6 +133,16 @@ def qflxconvert_units(var):
     return var
 
 
+def w_convert_q(var):
+    if var.units == "mol/mol":
+        var = (
+            var * 18.0 / 28.97 * 1000.0
```
Comment (chengzhuzhang) [inline e3sm_diags/derivations/acme.py:139]: I have updated the formula and comments. Thank you for the suggestion and review, @tangq

**Pull request merged**